<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutPrincipal">
     <item>
      <layout class="QGridLayout" name="layoutCalc" rowstretch="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="4">
        <widget class="QLineEdit" name="inputLine">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacerBasLineEdit">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <layout class="QGridLayout" name="layoutCalculesBasiques">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="buttonQuote">
           <property name="text">
            <string>'</string>
           </property>
           <property name="shortcut">
            <string>'</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="button7">
           <property name="text">
            <string>7</string>
           </property>
           <property name="shortcut">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="button4">
           <property name="text">
            <string>4</string>
           </property>
           <property name="shortcut">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="button1">
           <property name="text">
            <string>1</string>
           </property>
           <property name="shortcut">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="button0">
           <property name="text">
            <string>0</string>
           </property>
           <property name="shortcut">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="buttonSpace">
           <property name="text">
            <string>Space</string>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonDiv">
           <property name="text">
            <string>/</string>
           </property>
           <property name="shortcut">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="button8">
           <property name="text">
            <string>8</string>
           </property>
           <property name="shortcut">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="buttonMul">
           <property name="text">
            <string>*</string>
           </property>
           <property name="shortcut">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="buttonMoins">
           <property name="text">
            <string>-</string>
           </property>
           <property name="shortcut">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="button9">
           <property name="text">
            <string>9</string>
           </property>
           <property name="shortcut">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="buttonPlus">
           <property name="text">
            <string>+</string>
           </property>
           <property name="shortcut">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="button5">
           <property name="text">
            <string>5</string>
           </property>
           <property name="shortcut">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="button6">
           <property name="text">
            <string>6</string>
           </property>
           <property name="shortcut">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="button2">
           <property name="text">
            <string>2</string>
           </property>
           <property name="shortcut">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="button3">
           <property name="text">
            <string>3</string>
           </property>
           <property name="shortcut">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="buttonDot">
           <property name="text">
            <string>.</string>
           </property>
           <property name="shortcut">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" rowspan="3">
          <widget class="QPushButton" name="buttonEnter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="buttonSuppr">
           <property name="text">
            <string>Suppr</string>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="buttonDel">
           <property name="text">
            <string>Del</string>
           </property>
           <property name="shortcut">
            <string>Backspace</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="4">
          <layout class="QGridLayout" name="layoutCalculesScientifiques">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonInv">
             <property name="text">
              <string>INV</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="buttonSign">
             <property name="text">
              <string>SIGN</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="buttonModulo">
             <property name="text">
              <string>MOD</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="buttonComplexe">
             <property name="text">
              <string>$</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonCos">
             <property name="text">
              <string>COS</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="buttonSin">
             <property name="text">
              <string>SIN</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonTan">
             <property name="text">
              <string>TAN</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QCheckBox" name="checkBoxHyperbolique">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>H</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="buttonSqr">
             <property name="text">
              <string>SQR</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="buttonCube">
             <property name="text">
              <string>CUBE</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="buttonPow">
             <property name="text">
              <string>^</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="buttonSqrt">
             <property name="text">
              <string>SQRT</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="buttonLn">
             <property name="text">
              <string>LN</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="buttonLog">
             <property name="text">
              <string>LOG</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3" rowspan="2">
            <widget class="QPushButton" name="buttonEval">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>EVAL</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="layoutPile">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <spacer name="verticalSpacerHautPile">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QListView" name="listPile">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacerBasPile">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <layout class="QGridLayout" name="layoutFonctionPile">
         <item row="1" column="0">
          <widget class="QPushButton" name="buttonSwap">
           <property name="text">
            <string>SWAP</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonClearPile">
           <property name="text">
            <string>CLEAR</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="buttonSum">
           <property name="text">
            <string>SUM</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="buttonMean">
           <property name="text">
            <string>MEAN</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="buttonDup">
           <property name="text">
            <string>DUP</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="buttonDrop">
           <property name="text">
            <string>DROP</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="buttonAffiche">
           <property name="text">
            <string>Affiche N</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <widget class="QMenu" name="menuComplexe">
     <property name="title">
      <string>mode complexe</string>
     </property>
     <addaction name="actionYes"/>
     <addaction name="actionNo"/>
    </widget>
    <widget class="QMenu" name="menuType">
     <property name="title">
      <string>choix type</string>
     </property>
     <addaction name="actionInteger"/>
     <addaction name="actionFloat"/>
     <addaction name="actionRational"/>
    </widget>
    <widget class="QMenu" name="menuChoixdegre">
     <property name="title">
      <string>choix degrés</string>
     </property>
     <addaction name="actionRadian"/>
     <addaction name="actionDegre"/>
    </widget>
    <widget class="QMenu" name="menuUtilisationClavier">
     <property name="title">
      <string>utilisation clavier</string>
     </property>
     <addaction name="actionClavierOn"/>
     <addaction name="actionClavierOff"/>
    </widget>
    <addaction name="menuComplexe"/>
    <addaction name="menuType"/>
    <addaction name="menuChoixdegre"/>
    <addaction name="menuUtilisationClavier"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionYes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>oui</string>
   </property>
  </action>
  <action name="actionNo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>non</string>
   </property>
  </action>
  <action name="actionInteger">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>entier</string>
   </property>
  </action>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>réel</string>
   </property>
  </action>
  <action name="actionRational">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>rationnel</string>
   </property>
  </action>
  <action name="actionRadian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>radiant</string>
   </property>
  </action>
  <action name="actionDegre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>degré</string>
   </property>
  </action>
  <action name="actionClavierOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>oui</string>
   </property>
  </action>
  <action name="actionClavierOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>non</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
